types/testing-library__jest-dom/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for @testing-library/jest-dom 4.0
+// Project: https://github.com/testing-library/jest-dom
+// Definitions by: Damien Senger <https://github.com/hiwelo>
+//                 Łukasz Fiszer <https://github.com/lukaszfiszer>
+//                 Revath S Kumar <https://github.com/revathskumar>
+//                 Ernesto García <https://github.com/gnapse>
+//                 Weyert de Boer <https://github.com/weyert>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+declare namespace jest {
```

Comment:
This is not going to work with `import {toBeInTheDocument, toHaveClass} from '@testing-library/jest-dom'` as per https://github.com/testing-library/jest-dom#usage

Reply:
I am not sure how to support this alternative way of using the library with TypeScript. Do we want to support the not typical usage of the library? Any tips how this could be resolved?

Reply:
Please read https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module for details on how to write definitions that are available as module or global